test/cpp/util/test_credentials_provider.cc

```diff
@@ -19,19 +19,19 @@
 
 #include "test/cpp/util/test_credentials_provider.h"
 
+#include <gflags/gflags.h>
```

Comment:
Why did the order of these includes change? Was this done by one of the scripts?

Reply:
Hmm I'm not sure if it was done by my IDE for by one of the scripts. The newly formatted version seems more correct, but I'll revert it back to how it was in case it was like that for a reason.